docker/aliases.sh

```diff
@@ -0,0 +1,20 @@
+# This is a compilation of useful 'USER-DEFINED' aliases to use
+
+alias tmux_create='tmux new -s'        # Creates new tmux session.
+alias tmux_attach='tmux a -t'          # Attaches to an existing tmux session.
+alias     tmux_ls='tmux ls'            # Lists all of the existing tmux sessions.
+alias   tmux_kill="tmux kill-session -t " # Kill a specific Tmux session
```

Comment:
I noticed `Tmux` is used here in the comment but should that be lowercase like line 3? Not a huge thing but it caught my eye and just wanted to ask.

Reply:
Yes, that makes sense. Just update it in the last set of changes. Thanks for the feedback.